deployments/kubernetes/chart/reloader/Chart.yaml

```diff
@@ -3,8 +3,8 @@
 apiVersion: v1
 name: reloader
 description: Reloader chart that runs on kubernetes
-version: v0.0.95
-appVersion: v0.0.95
+version: v0.0.96
```

Comment:
@gciria Version and appVersion do not need to be updated manually. Can you please revert it back?

Reply:
Oh, sorry @faizanahmad055 
Reverted!